.github/workflows/ci.yml

```diff
@@ -109,6 +109,12 @@ jobs:
             echo "$HOME/.cargo/bin" >> $GITHUB_PATH
           fi
 
+      # Always print disk usage, even for jobs that don't free up disk space.
+      - name: print disk usage
+        run: |
+          echo "disk usage:"
+          df -h
```

Comment:
the same step is present at line 255.
Why do we need to duplicate it?

Reply:
So we have a before and after comparison. One thing I encountered when first investigating this is that I didn't know how much of the space was used up by the OS and pre-installed software.